mail_tracking/static/src/js/mail_tracking.js

```diff
@@ -0,0 +1,60 @@
+/* © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). */
+
+if (typeof window.jQuery === 'undefined') { throw new Error('Requires jQuery'); }
+(function ($, window, document) {
+    'use strict';
+
+    openerp.mail_tracking = function (instance) {
+        var _t = instance.web._t,
+            _lt = instance.web._lt;
+        var QWeb = instance.web.qweb;
+        var mail_orig = instance.mail;
+        var mail_inherit = function() {
+            instance.mail.MessageCommon.include({
+                init: function (parent, datasets, options) {
+                    this._super(parent, datasets, options);
+                    this.partner_trackings = datasets.partner_trackings || [];
+                }
+            });
+            instance.mail.ThreadMessage.include({
+                bind_events: function () {
+                    this._super();
+                    this.$('.oe_mail_action_tracking').on('click', this.on_tracking_status_clicked);
```

Comment:
But I'm sure you want [`this.do_action(action)`](https://github.com/OCA/social/pull/67/files/129ed1ba89cff54fcee0ab538f2c20093031cea5#diff-43f85605fe16954a8ada1881011a8bd1R44) to reach the method it's calling, and that's not in the DOM element but in the `ThreadMessage` object.